README.md

```diff
@@ -2,7 +2,9 @@
 
 #### Aws client as a Symfony Service
 
+# /!\ This bundle is deprecated
```

Comment:
you can make a real warning sign in emoji
# :warning: This bundle is deprecated